airflow/providers/amazon/aws/transfers/s3_to_redshift.py

```diff
@@ -87,13 +87,11 @@ def __init__(
         self.verify = verify
         self.copy_options = copy_options or []
         self.autocommit = autocommit
-        self._s3_hook = None
-        self._postgres_hook = None
 
-    def execute(self, context):
-        self._postgres_hook = PostgresHook(postgres_conn_id=self.redshift_conn_id)
-        self._s3_hook = S3Hook(aws_conn_id=self.aws_conn_id, verify=self.verify)
-        credentials = self._s3_hook.get_credentials()
+    def execute(self, context: Dict[str, Any]) -> None:
+        postgres_hook = PostgresHook(postgres_conn_id=self.redshift_conn_id)
```

Comment:
These variables are referenced from only `execute()`, so I changed scope of them. If we keep them as  the attributes of the instance, their type are Optional[T]. I think it's a little redundant, because they have always actual values in execute().